### DISCUSSION TITLE
fullwidth.html doesn't exist?

### DISCUSSION THREAD

**Discussion by jfmatth** [django-cms/django-cms#8003] [General]
Hi, trying to get django-cms working and the tutorial mentions that fullwidth.html is the first of my templates in settings.py but that file is NO WHERE To be found, do i make it?

The install for non-docker is very minimal and DOES not match the tutorial

Again, unless I'm not getting it?

**Comment by fsbraun** [comment]
It seems that `fullwidth.html` indeed is mentioned [here](https://docs.django-cms.org/en/latest/introduction/02-templates_placeholders.html). It will need to be changed. I've created an issue here: #8005.

Please read `minimal.html`(for the quickstart docker project found in `django-cms-quickstart/backend/templates`) instead. Its content is 
```
{% load cms_tags sekizai_tags %}
<html>
    <head>
        <title>{% page_attribute "page_title" %}</title>
        {% render_block "css" %}
    </head>
    <body>
        {% cms_toolbar %}
        {% placeholder "Content" %}
        {% render_block "js" %}
    </body>
</html>
```

The quickest way to get started probably is the [`djangocms` command](https://docs.django-cms.org/en/latest/introduction/01-install.html#installing-django-cms-by-hand). It is fully functional but minimal in the sense, that it does not define its own templates, or relies on sqlite as a database. But it is a good starting point to create your own project (just as `django-admin startproject` is just a starting point for your own project).

**Reply by jfmatth** [reply]
@fsbraun Thank you for the quick reply.  DjangoCMS looks like what I want, just having a tough time getting started.

The docker 'quickstart' is pretty deep if you're not into Docker-compose much, and the simple djangocms myproject is almost 180 in terms of what's included.  Be nice if the docker-compose version matched something without docker :).

now that I know fullwidth doesn't exist and minimal is the one, ill continue.

**Reply by fsbraun** [reply]
@jfmatth There is a learning curve, but I hope you'll grow to love django CMS. Be invited to join our Discord server and get quick help from the community: https://discord-support-channel.django-cms.org/

**Reply by jfmatth** [reply]
Thanks again for the quick response.  Ill get on the server next week. 

Looking to move from hugo to something a little more drag and drop for me.  No need to integrate to existing django apps (yet) :)

**Comment by jfmatth** [comment]
HI @fsbraun :)

I'm finding more docs errors throughout the 01-install.html file. 

Would you like a PR for what I find?

**Reply by fsbraun** [reply]
That would be wonderful!

**Comment by fsbraun** [comment]
Transfered to issue #8005
